### DISCUSSION TITLE
portable on computers without MSVC Support Libraries?

### DISCUSSION THREAD

**Discussion by janpeterd** [keepassxreboot/keepassxc#8486] [Q&A] [answered]
I want to have a USB-stick with my keepass-database and keepassxc portable, but it only runs on pc's with MSVC Support Libraries installed. I wonder if there is a way to have all dependencies on the USB, so I can access my passwords on any pc, even when I don't have the permissions to install software (like my schools pc).

**Comment by droidmonkey** [comment, marked as answer]
Use the legacy windows portable distribution.

**Reply by phoerious** [reply]
It should be enough to extract the DLLs and put them in the same directory.

**Reply by janpeterd** [reply]
Do you know where I can find these DLLs?

**Reply by phoerious** [reply]
After installation, C:\Windows\system32 I suppose, but you can also unpack the installer with `cabextract`, which should give you a bunch of ZIP archives containing the DLLs. But I cannot test it right now.
